c/driver/postgresql/statement.cc

```diff
@@ -206,15 +211,17 @@ struct BindStream {
           param_lengths[i] = 0;
           break;
         default:
-          SetError(error, "Field #", i + 1, " ('", bind_schema->children[i]->name,
+          SetError(error, "%s%" PRIu64 "%s%s%s%s", "[libpq] Field #",
```

Comment:
The R builds using mingw32 didn't like the %z specifier `statement.cc:214:31: warning: unknown conversion type character 'z' in format [-Wformat=]`